paddle/phi/api/yaml/ops.yaml

```diff
@@ -2035,7 +2035,7 @@
   backward : psroi_pool_grad
 
 - op : put_along_axis
-  args : (Tensor arr, Tensor indices, Tensor values, int axis, str reduce = "assign")
+  args : (Tensor arr, Tensor indices, Tensor values, int axis, str reduce = "assign", bool include_self = true)
```

Comment:
there is a parameter of `broadcast` in Python API, shall we also add it here as `include_self ` or delete it from API？

Reply:
`broadcast` is processed  in the Python interface, so there is no need to pass it into the C interface again